cpp/src/arrow/acero/asof_join_benchmark.cc

```diff
@@ -91,7 +91,7 @@ static void TableJoinOverhead(benchmark::State& state,
     ASSERT_OK(DeclarationToStatus(std::move(join_node), /*use_threads=*/false));
   }
 
-  state.counters["input_rows_per_second"] = benchmark::Counter(
+  state.counters["rows_per_second"] = benchmark::Counter(
```

Comment:
All the other codes also use the `rows_per_second` value, so I made the modifications accordingly.